diagnostic.md

```diff
@@ -57,13 +70,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li').parent().parent().hide();
 <!-- your answer ends here -->
 
 ## Question 8
 
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+$("div").children().on("click", function(){
+    alert("The paragraph was clicked.");
+});
```

Comment:
the child handler is correct, but please note that it should be on the `div` items, not their children